docs/layers/cpu-grid-layer.md

```diff
@@ -193,19 +193,14 @@ It takes an array of points inside each cell as arguments, returns a number. For
 You can pass in `getColorValue` to color the cells by avg/mean/max of a specific attributes of each point.
 By default `getColorValue` returns the length of the points array.
 
-Note: grid layer compares whether `getColorValue` has changed to recalculate the value for each bin that its color based on.
-You should pass in the function defined outside the render function so it doesn't create a new function on every rendering pass.
+`getColorValue` is compared using [`updateTriggers`](https://deck.gl/#/documentation/deckgl-api-reference/layers/layer?section=_datadiff-function-optional-experimental)
```

Comment:
- Wrong link
- Why do we need to call out `getColorValue` specifically? Can we just link to the [accessors section](https://github.com/uber/deck.gl/blob/master/docs/developer-guide/using-layers.md#accessors) like any other prop?

Reply:
i am no difference, previously we did call out `getColorValue` say that it is shallow comparison...Maybe should just add to upgrade guide instead

Reply:
+1 to upgrade guide. The way our docs are organized it is hard to track which version a change is introduced in :(